scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -15,7 +15,7 @@ workflow GvsExtractCallset {
     String extract_table_prefix
     String filter_set_name
     String query_project = project_id
-    Int scatter_count
+    Int? scatter_count
```

Comment:
do we want to provide a comment that no default is needed since it is calculated now during the workflow

Reply:
Sure I could do that here.